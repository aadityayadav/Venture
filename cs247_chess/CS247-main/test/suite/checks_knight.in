setup
+ K a1
+ N e4
+ k f2
done
- f2
+ k d2
done
- d2
+ k c3
done
- c3
+ k c5
done
- c5
+ k d6
done
- d6
+ k f6
done
- f6
+ k g5
done
- g5
+ k g3
done
- g3
+ k h8
done
