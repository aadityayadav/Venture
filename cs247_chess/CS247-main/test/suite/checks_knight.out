The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
_______k
________
________
________
____N___
________
________
K_______
